src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/testpluggables/TestReadFilter.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.hellbender.cmdline.GATKPlugin.testpluggables;
+
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
```

Comment:
Hmm... Is there an issue with putting these explicitly in the test package?

Reply:
They're in the test package specifically so they're not discoverable under normal circumstances.  They aren't found by the plugin unless you explicitly tell it to look in this package.